rust-features.sh

```diff
@@ -0,0 +1,96 @@
+#!//usr/bin/env bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```
Nope but it would still work :smile: